extras/ioc_scan_by_host.sh

```diff
@@ -50,15 +50,15 @@ echo Instances are: "${instances[*]}"
 # Loop through all instance IDs
 for instance_id in "${instances[@]}"; do
   echo
-  echo Searching "$instance_id":
+  echo -n Searching "$instance_id":
 
   # Use find-grep to search for IOC strings in log files, ignoring
   # *.journal files.  We pipe the result into another grep process
   # that uses the --invert-match grep flag to exclude matches (e.g.,
   # from sudo.log) that contain our grep command (e.g. sudo.log).
```

Comment:
Updated to reflect the change in functionality.

```suggestion
  # Use find-zgrep to search for IOC strings in log files (including
  # gzipped log files), ignoring *.journal files.  We pipe the result
  # into another grep process that uses the --invert-match grep flag
  # to exclude matches (e.g., from sudo.log) that contain our grep
  # command (e.g. sudo.log).
```